docs/GettingStarted.md

```diff
@@ -0,0 +1,135 @@
+
+```java
+implementation 'com.adobe.marketing.mobile:userprofile:2.0'
+implementation 'com.adobe.marketing.mobile:core:2.0'
+implementation 'com.adobe.marketing.mobile:identity:2.0'
+implementation 'com.adobe.marketing.mobile:signal:2.0'
+implementation 'com.adobe.marketing.mobile:lifecycle:2.0'
+```
+
+2. Next you'll need to import SDK libraries into your project and register them for initialization. Extensions are registered with Mobile Core so that they can dispatch and listen for events.
+
+### Java
+
+```java
+public class MainApp extends Application {
+    private static final String APP_ID = "";
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        MobileCore.setApplication(this);
+        MobileCore.setLogLevel(LoggingMode.VERBOSE);
+        MobileCore.configureWithAppID(APP_ID);
+
+        List<Class<? extends Extension>> extensions = new ArrayList<>();
+        extensions.add(Lifecycle.EXTENSION);
+        extensions.add(Signal.EXTENSION);
+        extensions.add(UserProfile.EXTENSION);
+        extensions.add(Identity.EXTENSION);
+        MobileCore.registerExtensions(extensions, o -> {
+            Log.d(LOG_TAG, "AEP Mobile SDK is initialized");
+        });
+    }
+}
+```
+
+### Kotlin
+
+```kotlin
+class MyApp : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        MobileCore.setApplication(this)
+        MobileCore.setLogLevel(LoggingMode.VERBOSE)
+        MobileCore.configureWithAppID("YOUR_APP_ID")
+
+        val extensions = listOf(Identity.EXTENSION, Signal.EXTENSION, Lifecycle.EXTENSION, UserProfile.EXTENSION)
+        MobileCore.registerExtensions(extensions) {
+            Log.d(LOG_TAG, "AEP Mobile SDK is initialized")
+        }
+    }
+
+}
+```
+
+## Implement Lifecycle Metrics
+
+Lifecycle metrics is an optional, yet valuable feature provided by the Adobe Experience Platform SDK. It offers out-of-the-box, application lifecycle information about an app user. These metrics contain information on the app user's engagement lifecycle, such as device information, install or upgrade information, session start and pause times, and more.
+
+Withint the `Activity.onResume()` function, start Lifecycle data collection:
+
+### Java
+
+```java
+@Override
+public void onResume() {
+    super.onResume();
+    MobileCore.setApplication(getApplication());
+    MobileCore.lifecycleStart(null);
+}
+```
+
+### Kotlin
+
+```kotlin
+override fun onResume() {
+    super.onResume()
+    MobileCore.setApplication(application)
+    MobileCore.lifecycleStart(null)
+}
+```
+
+Setting the application is only necessary on activities that are entry points for your application. However, setting the application on each Activity has no negative impact and ensures that the SDK always has the necessary reference to your application. As a result, you should call setApplication on each of your activities.
+You can use the onPause function to pause the lifecycle data collection:
```

Comment:
Can you make the api names to be in code format.
example : You can use the `onPause` ... , As a result, you should call `setApplication` ...